docs-chef-io/content/inspec/cli.md

```diff
@@ -116,6 +116,10 @@ This subcommand has the following additional options:
 `--no-with-cookstyle`
 : Enable or disable cookstyle checks.
 
+`--legacy-check`
+`--no-legacy-check`
+: For running checks on profiles that needs train connections, run `inspec check --legacy-check`. Default: do not check.
```

Comment:
"Run check in legacy mode, which examines the profile in a different way. Default: use current parser-based method."